language/tools/move-coverage/check_coverage.bash

```diff
@@ -14,14 +14,7 @@ pushd ../../../diem-move/diem-framework || exit 1
 cargo run
 popd || exit 1
 
-#cargo test -p ir-testsuite -p language-e2e-tests -p move-lang-functional-tests
-
-echo "---------------------------------------------------------------------------"
-echo "Running IR testsuite..."
-echo "---------------------------------------------------------------------------"
-pushd ../../ir-testsuite || exit 1
-cargo test
-popd || exit 1
+# TODO: add coverage for transactional tests
```

Comment:
Good TODO. I wonder if we ought to create a GitHub issue to track this now, while we still have some context here as to how these test suites used to be tested for coverage, but are no longer as part of the migration to the new test runner.

(What are your thoughts on GitHub issues in general? Part of me thinks we could use them more, to document our planned changes and motivation, and to potentially hand off work between one another. Another part of me thinks it might slow us down, writing issues that we won't end up re-reading or referencing again. What do you think?)

Reply:
> I wonder if we ought to create a GitHub issue to track this now, while we still have some context here as to how these test suites used to be tested for coverage, but are no longer as part of the migration to the new test runner.

Sounds reasonable in this case. I'll create one.

> What are your thoughts on GitHub issues in general?

I share a similar view as yours -- issues can be a hit or miss depending on the cases and how they are handled.